assets/src/components/FeaturesTable.js

```diff
@@ -0,0 +1,446 @@
+/**
+ * @module components/FeaturesTable.js
+ * @name FeaturesTable
+ * @copyright 2024 3Liz
+ * @author DOUCHIN Michaël
+ * @license MPL-2.0
+ */
+
+import { html, render } from 'lit-html';
+import { mainLizmap, mainEventDispatcher } from '../modules/Globals.js';
+
+/**
+ * @class
+ * @name FeaturesTable
+ * @augments HTMLElement
+ */
+export default class FeaturesTable extends HTMLElement {
+
+    constructor() {
+        super();
+
+        // Random element id
+        this.id = window.crypto.randomUUID();
+
+        // Layer name
+        this.layerTitle = this.getAttribute('layerTitle') || 'Features table: error';
+
+        // Layer id
+        this.layerId = this.getAttribute('layerId');
+
+        // Error text
+        this.error = null;
+
+        // Get the layer name & configuration
+        const layerName = lizMap.getLayerConfigById(this.layerId)?.[0];
+        let layerConfig = lizMap.getLayerConfigById(this.layerId)?.[1];
+        this.layerConfig = layerConfig;
```

Comment:
```suggestion
        // Get the layer name & configuration
        this.layerConfig = mainLizmap.initialConfig.layers.getLayerConfigByLayerId(this.layerId);
        const layerName = this.layerConfig.name;
```
If `this.layerId` is unknown, the `getLayerConfigByLayerId` will throw an error.
You can check if the llayerId is known with `mainLizmap.initialConfig.layers.layerIds.includes(this.layerId)`